src/top-bar/org-unit-select/org-unit-select.js

```diff
@@ -14,7 +14,7 @@ const OrgUnitSelect = () => {
         setOpenedSelect,
     } = useSelection()
```

Comment:
I was wondering what exactly this hook does, so I had to go to the file, it's `useContext` under the hood, so maybe rename this one to `useSelectionContext`? That makes it quite clear